templates/base.html

```diff
@@ -81,6 +81,9 @@
         <li class="nav-item">
           <a href="/meeting/notification_demo" class="nav-link">Notification Demo</a>
         </li>
+        <li class="nav-item">
+          <a href="/meeting/logout" class="nav-link">Logout</a>
+        </li>
```

Comment:
Heads up: This appears on the page, even if the user isn't logged in.

Reply:
That can be for next sprint possibly, but I don't think the base.html is pulled into any file accessible to the user without them being logged in.

Reply:
Yeah I didn't see it as a huge deal at all, this branch is merged.